modules/ROOT/pages/exportword.adoc

```diff
@@ -0,0 +1,62 @@
+= Export to Word
+:navtitle: exportword
+:description: The export to Word feature lets you generate a .docx file directly from the editor.
+:description_short: Generate a .docx file directly from the editor.
+:keywords: plugin, exportword.
+:pluginname: Export to Word
+// values for <tier>
+// opensource plugin = tieroneplan
+// premium plugin    = tiertwoplan or tierthreeplan or enterpriseplan
+:pluginminimumplan: <tier>
+:plugincode: exportword
+:plugincategory: premium
+
+include::partial$misc/admon-paid-addon-pricing.adoc[]
+
+include::partial$misc/admon-requires-7.0v.adoc[]
+
+The export to Microsoft Word feature collects the HTML generated with the `tinymce.editor.getContent()` method and the default editor content styles combined with the styles provided by you in the configuration. It then sends them to the {productname} Cloud Services HTML to DOCX converter service. The service generates a Word file and returns it to the user’s browser so they can save it in the Word format on their disk.
```

Comment:
Do we usually write docs in second person form and use `you` or should this be reformulated to be `styles provided by the integrator` or `custom defined styles` or something?

Reply:
```suggestion
The export to Microsoft Word feature collects the HTML generated with the `tinymce.editor.getContent()` method and combines it with the default editor content styles along with the styles provided in the configuration. The combined content and styles are then transmitted to the {productname} Cloud Services HTML to DOCX converter service. Following this, the service generates a Word file, which is subsequently returned to the user’s browser, enabling them to save it in the Word format onto their disk.
```